forge-game/src/main/java/forge/game/ability/SpellAbilityEffect.java

```diff
@@ -636,10 +636,8 @@ protected static boolean addToCombat(Card c, Player controller, SpellAbility sa,
             combat.initConstraints();
             if (sa.hasParam("ChoosePlayerOrPlaneswalker")) {
                 PlayerCollection defendingPlayers = AbilityUtils.getDefinedPlayers(sa.hasParam("ForEach") ? c : host, attacking, sa);
-                defs = new FCollection<>();
-                for (Player p : defendingPlayers) {
-                    defs.addAll(combat.getDefendersControlledBy(p));
-                }
+                defs = new FCollection<>(defendingPlayers);
+                defs.addAll(Iterables.filter(combat.getDefendingPlaneswalkers(), CardPredicates.isControlledByAnyOf(defendingPlayers)));
```

Comment:
isn't isControlledByAnyOf wrong? shouldn't that be Protector?

also shouldn't you make new MR?

Reply:
Myriad and the few other cards that use it should not interact with battles...
>attacking that player or a planeswalker they control